contracts/protocol/contracts/ERC1155/ScoutProtocolBuilderNFTImplementation.sol

```diff
@@ -436,6 +430,31 @@ contract ScoutProtocolBuilderNFTImplementation is
         return MemoryUtils._getAddress(MemoryUtils.MINTER_SLOT);
     }
 
+    function mintTo(
```

Comment:
new method mintTo() allows us to mint NFTs after an initial silent auction directly to auction winners